actions/docker-push/1.0/Dockerfile

```diff
@@ -1,18 +1,20 @@
-FROM --platform=$BUILDPLATFORM registry.erda.cloud/retag/gcrane:v0.15.1 as gcrane
-FROM --platform=$BUILDPLATFORM registry.erda.cloud/erda/terminus-golang:1.19.7 as builder
+FROM registry.erda.cloud/erda/terminus-golang:1.19.7 as builder
 
-ARG TARGETARCH
 
 COPY . /go/src/github.com/erda-project/erda-actions
 WORKDIR /go/src/github.com/erda-project/erda-actions
 
-ARG ARCH
-
 ENV GOPROXY=https://goproxy.cn,direct
 
-RUN GOOS=linux GOARCH=$TARGETARCH go build -o /opt/action/run github.com/erda-project/erda-actions/actions/docker-push/1.0/internal
+RUN go build -o /opt/action/run github.com/erda-project/erda-actions/actions/docker-push/1.0/internal
 
-FROM --platform=$BUILDPLATFORM registry.erda.cloud/retag/debian:buster-slim
+FROM registry.erda.cloud/erda-actions/custom-script:2.0
 
-COPY --from=gcrane /ko-app/gcrane /usr/bin
 COPY --from=builder /opt/action/run /opt/action/run
+
+RUN curl -L \
+    https://github.com/google/go-containerregistry/releases/latest/download/go-containerregistry_Linux_x86_64.tar.gz \
```

Comment:
multi arch

Reply:
adjusted, rebuild gcrane image and copy gcrane from retag image